src/compileTranslations.ts

```diff
@@ -73,12 +73,12 @@ export default function compileTranslations(defaultLocale: string) {
 
       if (
         languageTable.languages.size > 0 &&
-        !languageTable.languages.has(defaultLocale)
+        !languageTable.languages.has(defaultLanguage)
       ) {
         done(
           new PluginError(
             PLUGIN_NAME,
-            `No translation file found for default locale "${defaultLocale}"`,
+            `No translation file found for default language: ${defaultLanguage}`,
```

Comment:
Did you mean to remove the quotes? That makes it inconsistent with the other error messages.

Reply:
I meant to remove them in both parts of this PR to be consistent with other errors, eg those in the project config validation. Just noticed now the rest of this file uses quotes everywhere though, so I'll switch it back.